backend/chainlit/data/dynamodb.py

```diff
@@ -510,6 +539,7 @@ async def get_thread(self, thread_id: str) -> "Optional[ThreadDict]":
         steps = []
         elements = []
 
+        thread_items = self._convert_decimal_to_floats(thread_items)
```

Comment:
Can we include thse conversions in `_serialize_item` and `_deserialize_item` itself.